ols/app/models/config.py

```diff
@@ -418,6 +466,8 @@ def validate_yaml(self) -> None:
         if self.ols_config is None:
             raise InvalidConfigurationError("no OLSConfig found")
         self.ols_config.validate_yaml()
+        if self.dev_config:
```

Comment:
Seems the `if` is not needed if dev_config is always there. May be it doesn't need to be Optional?

Reply:
yeah this was a leftover from when i originally had dev_config as possible to be None, thanks for catching it.

Reply:
fixed